src/ServerlessAutoSwagger.ts

```diff
@@ -157,12 +157,6 @@ export default class ServerlessAutoSwagger {
 
             const definitions: Record<string, Definition> = JSON.parse(definitionsData).components.schemas;
 
-            if (data.includes('anyOf')) {
-              // anyOf caused some issues with certain swagger configs
-              console.log('includes anyOf');
-              // const newDef = Object.values(definition).map(recursiveFixAnyOf);
-            }
```

Comment:
Definitely agree this should be removed – can you add something like ``TODO: Handle `anyOf` in swagger configs`` just so this isn't completely lost to us?

Reply:
done )